ffi/capi/dart/package/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: icu
```

Comment:
Maybe call it `icu4x`? `icu` is already taken on pub.dev, but icu4x is not, and I think it's also a nicer name. (https://pub.dev/packages/icu4x)

Reply:
We would rather call it `icu`, the Rust crate is called that and the npm package will be called that as well. That package looks super abandoned, is there a way to take it over?

Reply:
I messaged the current owner. If this doesn't work, we would have to ask the pub team if this warrants an exception.